### ISSUE TITLE
Document sharing does not work with asset-pipeline enabled

### ISSUE THREAD

**Issue by sndrr** [owncloud/documents#543]
### Steps to reproduce
1. enable asset-pipeline in the config/config.php ( 'asset-pipeline.enabled' => true,)
2. share a document by link (with or without password does not matter)
3. open this link in a private window

### Expected behaviour
The shared document should be shown.

### Actual behaviour
I see a blank screen.

### Server configuration
**Operating system**:
Debian Wheezy

**Web server:**
Apache/2.2.22 (Debian) Phusion_Passenger/3.0.13 PHP/5.4.44-0+deb7u1 mod_ssl/2.2.22 OpenSSL/1.0.1e

**Database:**
mysql  Ver 14.14 Distrib 5.5.44, for debian-linux-gnu (x86_64) using readline 6.2

**PHP version:**
PHP 5.4.44-0+deb7u1

**ownCloud version:** (see ownCloud admin page)
8.1.1.3

**Updated from an older ownCloud or fresh install:**
Updated

**List of activated apps:**
Files, Activity, Documents, Pictures, News, Contacts, Calendar
```
Enabled:
 - activity: 2.0.1
 - calendar: 0.7.2
 - contacts: 0.4.0.0
 - documents: 0.10.1
 - encryption: 1.0.0
 - files: 1.1.10
 - files_external: 0.2.3
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.2
 - files_texteditor: 0.4
 - files_trashbin: 0.6.3
 - files_versions: 1.0.6
 - files_videoviewer: 0.1.3
 - firstrunwizard: 1.1
 - gallery: 0.6.0
 - news: 5.3.9
 - templateeditor: 0.1
Disabled:
 - external
 - files_locking
 - provisioning_api
 - user_external
 - user_ldap
 - user_webdavauth
```

**The content of config/config.php:**

```
<?php
$CONFIG = array (
  'instanceid' => '5268235533ee8',
  'passwordsalt' => '********',
  'datadirectory' => '/home/oc/data',
  'dbtype' => 'mysql',
  'dbname' => 'owncloud',
  'dbuser' => 'owncloud',
  'dbpassword' => '********',
  'dbhost' => '127.0.0.1',
  'dbtableprefix' => 'oc_',
  'version' => '8.1.1.3',
  'installed' => true,
  'forcessl' => true,
  'maxZipInputSize' => 838860800,
  'allowZipDownload' => true,
  'theme' => '',
  'maintenance' => false,
  'loglevel' => '0',
  'trusted_domains' =>
  array (
    0 => 'redacted',
    1 => 'redacted',
  ),
  'preview_libreoffice_path' => '/usr/bin/libreoffice',
  'mail_from_address' => 'owncloud',
  'mail_smtpmode' => 'php',
  'mail_domain' => 'redacted',
  'secret' => 'redacted',
  'asset-pipeline.enabled' => true,
);
```

**Are you using external storage, if yes which one:** no

**Are you using encryption:** yes

### Client configuration
**Browser:**
Firefox 40.0.3

**Operating system:**
Windows

#### Web server error log
```
[Tue Sep 01 11:26:49 2015] [error] [client 92.63.171.239] File does not exist: /home/oc/apps/documents/css/3rdparty/webodf/editor.css, referer: https://website/public.php?service=documents&t=v6brMM2kRZoSHac
```

#### ownCloud log (data/owncloud.log)
```
{"reqId":"8lr0WsHrMegj3GPmC9pO","remoteAddr":"92.63.171.239","app":"documents","message":"Could not find resource file \"\/apps\/documents\/css\/3rdparty\/webodf\/editor.css\"","level":3,"time":"2015-09-01T09:27:23+00:00","method":"GET","url":"\/public.php?service=documents&t=v6brMM2kRZoSHac"}
{"reqId":"8lr0WsHrMegj3GPmC9pO","remoteAddr":"92.63.171.239","app":"remote","message":"The source file \"\/home\/oc\/apps\/documents\/css\/3rdparty\/webodf\/editor.css\" does not exist.","level":4,"time":"2015-09-01T09:27:25+00:00","method":"GET","url":"\/public.php?service=documents&t=v6brMM2kRZoSHac"}
{"reqId":"8lr0WsHrMegj3GPmC9pO","remoteAddr":"92.63.171.239","app":"documents","message":"Could not find resource file \"\/apps\/documents\/css\/3rdparty\/webodf\/editor.css\"","level":3,"time":"2015-09-01T09:27:25+00:00","method":"GET","url":"\/public.php?service=documents&t=v6brMM2kRZoSHac"}
{"reqId":"8lr0WsHrMegj3GPmC9pO","remoteAddr":"92.63.171.239","app":"PHP","message":"RuntimeException: The source file \"\/home\/oc\/apps\/documents\/css\/3rdparty\/webodf\/editor.css\" does not exist. at \/home\/oc\/3rdparty\/kriswallsmith\/assetic\/src\/Assetic\/Asset\/FileAsset.php#62","level":3,"time":"2015-09-01T09:27:27+00:00","method":"GET","url":"\/public.php?service=documents&t=v6brMM2kRZoSHac"}
```

#### Browser log
I don't think they're needed, since it's a simple 404.

#### Workaround
Disable asset-pipeline.

**Comment by sndrr** [comment]
As indicated, there is a workaround: don't enable asset-pipeline, but that comes at a performance cost.

**Comment by VicDeo** [comment]
@sndrr I've just published v0.10.2 at appstore.
I guess https://github.com/owncloud/documents/commit/9e5c22e333fa3e9448687f12165d3de69933fa83 fixing this issue. Please let me know if it is not.

**Comment by sndrr** [comment]
@VicDeo Yes, this fixes the issue.
